### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by weiji14**
Starting a CONTRIBUTING.md that includes ways people can contribute to the project. This includes discussions, issues, pull requests and translations (in the future). Detailed step by step instructions on opening a Pull Request from the GitHub user interface. Lots of these were adapted from https://github.com/github/docs/blob/main/contributing/types-of-contributions.md

Addresses #9.

### CODE REVIEW

**Comment by AdrienWehrle** [comment]
Super nice work @weiji14 ! Thanks a lot! :smile:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+2. Navigate to the file you want to modify, e.g. the
+   [Sentinel-2 JSON file](https://github.com/APECS-Earth-Observation/Polar-EO-Database/blob/main/data/Sentinel-2.json).
+
+3. Click on the pen :pen: icon on the top right corner that says "Edit this file"
+
+4. This should bring you to a page like
+   https://github.com/APECS-Earth-Observation/Polar-EO-Database/edit/main/data/Sentinel-2.json
+   where you can make edits to the text using a web-based editor.
+   Feel free to switch between the "Edit file" and "Preview changes" tabs as
+   you modify the content to make sure things look ok.
```
[the remark this answers is not in the document]
Comment (AdrienWehrle) [reply CONTRIBUTING.md:51]: There is no need to fork. The patch is created automatically on commit!

[the remark this answers is not in the document]
Comment (AdrienWehrle) [reply CONTRIBUTING.md:51]: So it's literally commit + PR only!

[the remark this answers is not in the document]
Comment (AdrienWehrle) [reply CONTRIBUTING.md:51]: You can have a look at the video I sent on the slack to see how do things look like on the contributor side! :smile:

**Comment by weiji14** [comment]
Let's keep this PR open until after the meeting, in case anyone has any feedback.

**Comment by AdrienWehrle** [comment]
I'm not sure the screen recording of the PR process would add a lot, your description is very clear and already includes screenshots! What do you think?

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (weiji14) [reply CONTRIBUTING.md:51]: Hmm, it looks like there are some little differences in some of those points for when a new user makes a change. I'll look at your video closer and try to edit the steps later.

**Comment by weiji14** [comment]
> I'm not sure the screen recording of the PR process would add a lot, your description is very clear and already includes screenshots! What do you think?

Still think a screen recording is good to have, people learn in different ways (video/text) so we should have both!

**Comment by AdrienWehrle** [comment]
Ok! Let's include it in another PR!

**Pull request merged**